### PR TITLE
feat(backend): update units summary model

### PR DESCRIPTION
**Pull request by pbn4**
## Issue

- Closes #issue
- Addresses #986 
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Please include a summary of the change and which issue(s) is addressed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/addresses technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

Provide instructions so we can review.

Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes
- [ ] I have run `yarn generate:client` if I made backend changes
- [ ] I have exported any new pieces in ui-components

### CODE REVIEW

**Review by seanmalbert — CHANGES_REQUESTED** [review]
@pbn4 ,
This looks good. I resolved the conflicts and ran `generate:client`. Can you make sure the backend e2e tests are passing. Since this does cause breaking changes with the frontend, I think we'll want the frontend work to branch off of this and we'll get them all in together.

File: backend/core/src/units-summary/entities/units-summary-ami-level.entity.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+@Entity({ name: "units_summary_ami_levels" })
+export class UnitsSummaryAmiLevel {
+  @PrimaryGeneratedColumn("uuid")
+  @Expose()
+  @IsUUID(4, { groups: [ValidationsGroupsEnum.default] })
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  id: string
+
+  @ManyToOne(() => AmiChart, { eager: false, nullable: true })
+  amiChart?: AmiChart | null
```
Comment (emilyjablonski) [inline backend/core/src/units-summary/entities/units-summary-ami-level.entity.ts:18]: Why do we have amiChart and amiChartId?

**Comment by emilyjablonski** [comment]
@seanmalbert Can we consider renaming `unit` to `unitGroup` ?

**Comment by seanmalbert** [comment]
> @seanmalbert Can we remove `units` from a listing now that we have `unitsSummary` ?

@emilyjablonski , just noticed this, and I think we already talked about it, but yes.

**Review by seanmalbert — APPROVED** [review]
waiting on a couple more to get merged into this before merging

**Pull request merged**